integration-cli/docker_cli_build_test.go

```diff
@@ -2293,6 +2293,29 @@ func TestBuildExposeOrder(t *testing.T) {
 	logDone("build - expose order")
 }
 
+func TestBuildExposeHostPort(t *testing.T) {
```

Comment:
Do me a favor, make the changes that you think are correct (to check for the warning), and then commit/push them - I'd like to see the edits and I'd like to see what jenkins does.  It sure looks like its something specific to your setup.